src/main/java/it/pn/frontend/e2e/common/DettaglioNotificaSection.java

```diff
@@ -27,6 +27,8 @@ public class DettaglioNotificaSection extends BasePage {
     @FindBy(xpath = "//*[contains(@class, 'MuiTimelineItem-root')]")
     List<WebElement> tuttiStatiNotificaList;
 
+
+
```

Comment:
Extra space is unwanted. Please format all the files to avoid any formatting problems